packages/trader/src/Modules/CFD/Constants/cfd-specifications.js

```diff
@@ -1,68 +1,66 @@
-import { localize } from '@deriv/translations';
-
 const mt5 = {
     real_synthetic_specs: {
-        [localize('Leverage')]: localize('Up to 1:1000'),
-        [localize('Margin call')]: localize('100%'),
-        [localize('Stop out level')]: localize('50%'),
-        [localize('Number of assets')]: localize('20+'),
+        Leverage: 'Up to 1:1000',
```

Comment:
This will remove the translations. Maybe we can use localize in function to cause it to rerender without reloading

```
real_synthetic_specs: {
        [localize('Leverage')]: () => localize('Up to 1:1000'),
        ...
 }
 ```